data/custom_middleware/allCaps.js

```diff
@@ -0,0 +1,11 @@
+function allCapsPOST(req, res, next) {
+  req.body.name = req.body.name.toUpperCase();
+  next();
+}
+
+function allCapsPUT(req, res, next) {
+  req.body.changes.name = req.body.changes.name.toUpperCase();
+  next();
+}
+
+module.exports = { allCapsPOST, allCapsPUT };
```

Comment:
nice consideration for implementing a different middleware for each case here -- now an extra challenge (totally fine the way it is for MVP): instead of making the name ALL CAPS only capitalize the first letter, and for _each word_ in the name-- so `chris foster` gets passed to next as Chris Foster instead of CHRIS FOSTER